app/models/bgs_dependents/veteran.rb

```diff
@@ -52,5 +52,15 @@ def veteran_response(participant, va_file_number, address, end_product)
         benefit_claim_type_end_product: end_product
       }
     end
+
+    private
+
+    def marital_status(dependents_application)
+      spouse_lives_with_vet = dependents_application.dig('does_live_with_spouse', 'spouse_does_live_with_veteran')
+
+      return nil if spouse_lives_with_vet.nil?
+
+      spouse_lives_with_vet ? 'Married' : 'Separated'
+    end
```

Comment:
What about the never married guard clause? Looks like you blew it away when addressing [my earlier comment](https://github.com/department-of-veterans-affairs/vets-api/pull/4619#discussion_r466554173).
```ruby
return 'Never Married' if dependents_application.dig('veteran_was_married_before') == false
```

Reply:
Is it concerning that blowing away the guard clause didn't cause any specs to fail? 😬

Reply:
I am concerned about that too 😆 . Working on that context.